mock/data.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Should you really have this file under versioned? This file should be git ignored (as its generated by your json concat)

Reply:
Generated files typically are not versioned. 
Thus it might be a good idea to add this file as .gitignore ?

Reply:
will address later